spec/features/bulk_screen_spec.rb

```diff
@@ -0,0 +1,43 @@
+require 'spec_helper'
+
+# Feature/view tests for the (old) bulk actions view.
+feature 'Bulk actions view', js: true do
+  before :each do
+    @current_user = double(
+      :webauth_user,
+      login: 'sunetid',
+      logged_in?: true,
+      permitted_apos: [],
+      is_admin: true,
+      permitted_collections: [["string"]]
+    )
+
+    expect_any_instance_of(ApplicationController).to receive(:current_user).exactly(8).times.and_return(@current_user)
```

Comment:
This is way too specific and it's not really the point of this spec to check how many times a controller is called or not.